src/checkPublication.ts

```diff
@@ -176,7 +176,10 @@ export function checkPublication(
     allLogs: Array<AWS.CloudWatchLogs.OutputLogEvent>
   ): Promise<void> => {
     const errorRegexp = /WARN|ERROR|FATAL/;
-    const errors = allLogs.filter(log => errorRegexp.test(log.message)).map(log => log.message);
+    const errors = allLogs
+        .filter(log => errorRegexp.test(log.message))
+        .filter(log => log.message.startsWith("WARNING: sun.reflect.Reflection.getCallerClass is not supported"))
```

Comment:
seems like a sensible solution. Might be worth just putting a commenting that links to: https://issues.apache.org/jira/browse/LOG4J2-2537

Reply:
is the logic correct though? Should it be:

```
.filter(log => !log.message.startsWith("WARNING: sun.reflect.Reflection.getCallerClass is not supported"))
```

maybe I am getting confused.

Reply:
no you're right! Good catch.  I was thinking of putting a test in for it, but given that there is no test framework configured I thought it might be a bit much